drivers/watchdog/Kconfig.w91

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+# Telink W91 WDT configuration options
+config WDT_TELINK_W91
+	bool "Telink W91 series Watchdog (WDT) Driver"
+	default y
+	depends on DT_HAS_TELINK_W91_WATCHDOG_ENABLED
+	help
+	  Enable WDT driver for Telink W91-series MCUs.
+
+if WDT_TELINK_W91
+
+config WDT_TELINK_W91_IPC_RESPONSE_TIMEOUT_MS
```

Comment:
I think it should use common timeout, see #275

Reply:
Got your point, will change

Reply:
just changed